dev-docs/source/FlowchartCreation.rst

```diff
@@ -14,11 +14,11 @@ The flowchart diagrams are created by writing ``graphviz`` .dot files that descr
 
    .. diagram:: AlgorithmName-v1_wkflw.dot
 
-Examples of this can be found in `ReflectometryReductionOne1.rst <https://raw.githubusercontent.com/mantidproject/mantid/master/docs/source/algorithms/ReflectometryReductionOne-v1.rst>`__ and `ReflectometryReductionOneAuto-v1.rst <https://raw.githubusercontent.com/mantidproject/mantid/master/docs/source/algorithms/ReflectometryReductionOneAuto-v1.rst>`__.
+Examples of this can be found in `ReflectometryReductionOne-v2.rst <https://raw.githubusercontent.com/mantidproject/mantid/main/docs/source/algorithms/ReflectometryReductionOne-v2.rst>`__ and `ReflectometryReductionOneAuto-v1.rst <https://raw.githubusercontent.com/mantidproject/mantid/main/docs/source/algorithms/ReflectometryReductionOneAuto-v2.rst>`__.
```

Comment:
```suggestion
Examples of this can be found in `ReflectometryReductionOne-v2.rst <https://raw.githubusercontent.com/mantidproject/mantid/main/docs/source/algorithms/ReflectometryReductionOne-v2.rst>`__ and `ReflectometryReductionOneAuto-v2.rst <https://raw.githubusercontent.com/mantidproject/mantid/main/docs/source/algorithms/ReflectometryReductionOneAuto-v2.rst>`__.
```